### DISCUSSION TITLE
Download size and/or time estimate for containers.

### DISCUSSION THREAD

**Discussion by TomEmotion** [neurodesk/neurodesk.github.io#142] [Ideas]
Right now 40 minutes into a download of AFNI and it's nowhere near done. Would be useful to have an estimate of the overall size of the download and overall progress. This is especially the case for users in regions less well served by fast or unlimited internet.

**Comment by stebo85** [comment]
Dear @TomEmotion, I have an idea how to do this and will get to it after the break.

**Comment by stebo85** [comment]
Dear @TomEmotion ,

I added two mirror servers of our images (europe + US) which speeds up the download and prevents the 40 minute wait you where seeing (nectar cloud was down that day). I also added an ETA and percent downloaded:
![image](https://user-images.githubusercontent.com/4021595/103590084-acc0a000-4f38-11eb-84b3-45c4e30e76ad.png)

can you please test in the new version and report back?
docker run --privileged --name vnm -v ~/vnm:/vnm -e USER=neuro -p 6080:80 -p 5900:5900 vnmd/vnm:20210104
